src/test/util/event.rs

```diff
@@ -183,6 +183,24 @@ impl EventClient {
         }
         panic!("could not find event for {} command", command_name);
     }
+
+    /// Gets all of the command started events for a specified command name.
+    pub fn get_command_started_events(&self, command_name: &str) -> Vec<CommandStartedEvent> {
```

Comment:
This pattern has come up in a few tests I've written, so I factored it out into a method on `EventClient` and refactored a test I wrote for an option I previously implemented to use it.